projects/components/src/table/header/table-header-cell-renderer.component.ts

```diff
@@ -64,6 +76,10 @@ import { TableColumnConfigExtended } from '../table.service';
             </div>
           </ht-popover-content>
         </ht-popover>
+
+        <ng-template #htmlTooltip>
+          <div [innerHTML]="this.sanitizedHtmlForTooltip"></div>
```

Comment:
So I mentioned sanitization, but now that I think about it, angular should be doing that for us. Could you try removing the sanitizer and verifying that a tooltip is not executed (but still rendered?)

Ref: https://angular.io/guide/security#sanitization-example
Specifically:
> Angular recognizes the value as unsafe and automatically sanitizes it, which removes the onerror attribute but keeps safe content such as the <b> element.

Reply:
aah. cool. it does do the work for us.

Reply:
Updated.